bin/create-env-secrets.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+echo "====="
+echo "Welcome to the environment secrets setter!"
+echo "====="
+echo "Please select the target environment for the secrets:"
+echo "IMPORTANT: Ensure you are authenticated with the"
+echo "AWS CLI with the appropriate account,"
+echo "and using the correct profile."
+
+PS3='Please enter your choice: '
+options=("UAT" "SIT" "PROD" "Quit")
+select opt in "${options[@]}"
+do
+    case $opt in
+        "UAT")
+            ENV="uat"
+            break
+            ;;
+        "SIT")
+            ENV="sit"
+            break
+            ;;
+        "PROD")
+            ENV="prod"
+            break
+            ;;
+        "Quit")
+            exit
+            ;;
+        *) echo "invalid option $REPLY";;
+    esac
+done
+echo "You have selected the $ENV environment!"
+
+secrets_description=(
+    "csdap-client-id::The CSDAP client id."
+    "csdap-client-password::The CSDAP client password."
+    "csdap-host-url::The CSDAP host url."
+)    
+
+echo ""
+echo "===="
+echo ""
+
+for secret in "${secrets_description[@]}"
+do
+    secret_name="${secret%%::*}"
+    secret_description="${secret##*::}"
+    secret_full_path="csdap-cumulus/$ENV/$secret_name"
```

Comment:
We can drop the `csdap-` prefix and just use `cumulus` as the app prefix (i.e., `cumulus/$ENV/$secret_name`)